wordpress-mobile/WordPress-iOS
# Improve plugin removal

This adds better error handling when there's a failure removing a plugin and allows for active plugins to be removed.

Fixes #8373 

To test:

- Verify that the option to remove a plugin is available for active and inactive plugins.
- Verify that removing an active plugin works.
